### ISSUE TITLE
Initializing Legacy Material Support. Unless you have legacy plugins and/or data this is a bug!

### ISSUE THREAD

**Issue by Draghonir** [SkriptLang/Skript#6968]
### Skript/Server Version

```markdown
[18:41:37 INFO]: [Skript] Server Version: 1.21.1-3-70dfa46 (MC: 1.21.1)
[18:41:37 INFO]: [Skript] Skript Version: 2.9.1 (skriptlang-github)
[18:41:37 INFO]: [Skript] Installed Skript Addons:
[18:41:37 INFO]: [Skript] - skript-gui v1.3 (https://github.com/APickledWalrus/skript-gui)
[18:41:37 INFO]: [Skript] - Skore v2.0.6
[18:41:37 INFO]: [Skript] - SkBee v3.5.9 (https://github.com/ShaneBeee/SkBee)
[18:41:37 INFO]: [Skript] Installed dependencies:
[18:41:37 INFO]: [Skript] - Vault v1.7.3-b131
[18:41:37 INFO]: [Skript] - WorldGuard v7.0.11-SNAPSHOT+2287-8ffc81a
```

### Bug Description

An error message occasionally appears, causing the server to freeze for a few seconds. The TPS drops, and it seems like the server is blocking a Skript tick. Please check the error logs or screenshots.

### Expected Behavior

The server becomes unresponsive for 10 seconds, causing a TPS drop.

### Steps to Reproduce

I'm not sure how it happens; it occurs randomly without any particular action from the players.

### Errors or Screenshots

Error when server block : https://pastes.dev/CQDuPTBwZB
starting serveur no error : https://pastes.dev/MZ7nZ9nc7J
![image](https://github.com/user-attachments/assets/21edca65-82fe-4e26-95ce-0e375ae77158)

### Agreement

- [X] I have read the guidelines above and affirm I am following them with this report.

**Comment by sovdeeth** [comment]
This seems like something that should normally happen on startup rather than during runtime. Can you confirm it only occurs once per restart of the server?

**Comment by Fusezion** [comment]
I'm aware of the fact that `color of` expression also at times can cause legacy materials to be loaded, I only found this out recently. There's likely a handful of things that do this in skript, as legacy shouldn't be loaded until attempting to check/use them.

**Comment by sovdeeth** [comment]
Yes, though those things should be removed as Skript doesn't actually support legacy versions anymore.

**Comment by Draghonir** [comment]
> This seems like something that should normally happen on startup rather than during runtime. Can you confirm it only occurs once per restart of the server?

Thank you for your response.

To clarify, the issue does not occur during the server startup. Everything runs smoothly when the server restarts. However, the problem happens while the server is running, about 2 to 3 times within a 24-hour period.

When it occurs, the server freezes for a few seconds, causing a TPS drop.

**Comment by sovdeeth** [comment]
hmm multiple times per restart is odd. It'll probably get fixed over time as we remove legacy code that was meant for <1.12 support

**Comment by Draghonir** [comment]
I understand that this might be due to legacy code, and I'll keep an eye out for any updates that may resolve this issue. In the meantime, if there's anything I can do to minimize the impact or any temporary workaround, please let me know.

**Comment by sovdeeth** [comment]
You can avoid the bucket events

**Comment by Draghonir** [comment]
We don’t use bucket events in our scripts.
